apps/dashboard/app/assets/javascripts/apps.js.coffee

```diff
@@ -1,3 +1,6 @@
 # Place all the behaviors and hooks related to the matching controller here.
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/
+
+$(document).on 'ready turbolinks:load', ->
+  $('#all-apps-table').DataTable()
```

Comment:
We aren't using turoblinks yet but if we enabled it this code will execute twice on first page load - is that a problem?

Reply:
Yea good catch, with turbolinks enabled this refreshes, so it doesn't save the sorts orders or other options.

Reply:
Actually this line of code executes on every page.

Simple solution is to just place a `<script>` tag block after the creation of the `#all-apps-table` dom elements and place that on the all apps page.

That would actually address both issues.

Reply:
I got it working with this if block.  I'm wondering if adding `<script>` blocks to our views is an anti-pattern with web-packer?

```
  if($('#all-apps-table').length != 0)
    $('#all-apps-table').DataTable()
```

Reply:
So that still doesn't address the Turbolinks double load. I'm not ready to call a script block an anti-pattern. It is definitely convenient in many cases. I think it depends on what web-packer might do differently. 

A better pattern might be to reduce the size of the script blocks by moving sections of code into functions that webpacker exposes globally (in some fashion) so it can be used on a page in a `<script>` section. In this case its 1 line though.